### DISCUSSION TITLE
Integration of LWJGL?

### DISCUSSION THREAD

**Discussion by chengenzhao** [AlmasB/FXGL#1112] [Ideas]
Hi:

Seems like LWJGL is integrating to JavaFX for providing binding of native shader methods
[Status of DriftFX integration into LWJGL3](https://github.com/eclipse-efx/efxclipse-drift/issues/17)
so is it possible to provide a component of FXGL to use LWJGL and DriftFX to have wrapped shader method?

Cheers

**Comment by AlmasB** [comment]
I don't have any experience in DriftFX (and only very briefly touched LWJGL), so my answer is I don't know. If there is a minimal example of JavaFX working with the above, then I suppose one can build something similar.
